### ISSUE TITLE
Juice theme author has added an ad on the default template

### ISSUE THREAD

**Issue by ITSecUK** [Keats/gutenberg#2678]
# Documentation issue

## Summary
The documentation lists various themes, including "Juice". I installed this theme, and was shown an ad.

https://github.com/huhu/juice/blob/c6ad1fbe1c6298dc983f56a78d26ad460993e6a1/templates/index.html#L111

![image](https://github.com/user-attachments/assets/a1728cd7-4d11-4581-aefb-fe05fe4728e9)

I just wanted to highlight this as it is very concerning for a new user trying Zola for the first time as I could never trust the theme going forward, and now wonder about the rest of the themes. Of course I can remove this, but it does not seem in keeping with the ethos of the project.

## Proposed solution
Provide clear guidance within the theme documentation section about what the theme author is permitted to do while staying listed as a promoted theme. If it said clearly "theme authors are allowed to include ads or third party tracking" that would atleast allow new users to make an informed decision.

**Comment by Keats** [comment]
Yeah it's not mentioned anywhere on the README either cc @huhu

**Comment by ITSecUK** [comment]
@Keats I propose you remove the theme from the list. Doesn't show Zola in a good light for a default. I absolutely love zola btw, been using it all week!

**Comment by Keats** [comment]
Yes I was giving the author some days to change it while I am on holiday.
I'll remove it when I get back if it hasn't changed

**Comment by afranke** [comment]
It hasn’t changed and the theme is still listed.

**Comment by cestef** [comment]
bump

![Image](https://github.com/user-attachments/assets/724212e8-7036-4c98-8e4e-4fc5540803f1)

**Comment by Keats** [comment]
ah I've removed it from the repo but another theme is failing and blocking the deploy
